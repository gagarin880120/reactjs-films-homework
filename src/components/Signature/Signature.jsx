import { hot } from 'react-hot-loader/root';
import React from 'react';
import PropTypes from 'prop-types';
import styles from './signature.module.scss';

function Signature(props) {
  return (
    <header className={styles.header}>
      <h1>{props.name}</h1>
    </header>
  )
}

export default hot(Signature);

Signature.propTypes = {
  name: PropTypes.string
}
