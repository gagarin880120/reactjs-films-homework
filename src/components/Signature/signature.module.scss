.header {
  background-color: #3a3838;
  padding: 2% 0;

  h1 {
    font-size: 28px;
    text-align: center;
    color: #fff;
  }
}
