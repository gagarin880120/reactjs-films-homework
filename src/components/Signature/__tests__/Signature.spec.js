import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Signature from '../Signature';

test('Signature component should render without crashing', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Signature name="Andrei Mandryk" />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
