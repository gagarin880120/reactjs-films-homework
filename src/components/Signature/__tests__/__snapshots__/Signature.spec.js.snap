// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signature component should render without crashing 1`] = `
<header
  className="header"
>
  <h1>
    Andrei Mandryk
  </h1>
</header>
`;
